api/src/database/migrations/20191112124521_initial.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+export async function up(knex: Knex): Promise<any> {
+    await knex.schema.createTable('projects', (table) => {
+        table.uuid('id').primary();
+        table.string('project_name', 256).notNullable();
+        table.string('org_url', 2083);
```

Comment:
2083 seems like an odd value. 'sup with that?

As a rule "magic numbers" are discouraged as they're hard to read and, as here, tend to appear in multiple places (i.e. here and line 10) and then you change it in one place but meant to change it everywhere. Prefer something like:

```
var _MAX_URL_SIZE = 2083;

...

table.string('org_url', _MAX_URL_SIZE)
```